File: gilmorales/src/main/java/vistas/VistaVerPublicacionesGustadasUsuarioRegistrado.java
package vistas;

import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.littemplate.LitTemplate;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.template.Id;
import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
import com.vaadin.flow.component.html.H4;
import com.vaadin.flow.component.button.Button;
import com.vaadin.flow.component.html.H3;
import com.vaadin.flow.component.html.Label;

/**
 * A Designer generated component for the vista-ver-publicaciones-gustadas-usuario-registrado template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("vista-ver-publicaciones-gustadas-usuario-registrado")
@JsModule("./src/vista-ver-publicaciones-gustadas-usuario-registrado.ts")
public class VistaVerPublicacionesGustadasUsuarioRegistrado extends LitTemplate {

    

	@Id("layoutPublicacionesGustadas")
	private Element layoutPublicacionesGustadas;
	

	public Element getLayoutPublicacionesGustadas() {
		return layoutPublicacionesGustadas;
	}


	public void setLayoutPublicacionesGustadas(Element layoutPublicacionesGustadas) {
		this.layoutPublicacionesGustadas = layoutPublicacionesGustadas;
	}


	/**
     * Creates a new VistaVerPublicacionesGustadasUsuarioRegistrado.
     */
    public VistaVerPublicacionesGustadasUsuarioRegistrado() {
        // You can initialise any data required for the connected UI components here.
    }

}

File: gilmorales/frontend/src/vista-ver-publicaciones-gustadas-usuario-registrado.ts
import { LitElement, html, css, customElement } from 'lit-element';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';

@customElement('vista-ver-publicaciones-gustadas-usuario-registrado')
export class VistaVerPublicacionesGustadasUsuarioRegistrado extends LitElement {
  static get styles() {
    return css`
      :host {
          display: block;
          height: 100%;
      }
      `;
  }

  render() {
    return html`
<vaadin-vertical-layout id="layoutPublicacionesGustadas" style="width: 100%; height: 100%;"></vaadin-vertical-layout>
`;
  }


  // Remove this method to render the contents of this view inside Shadow DOM
  createRenderRoot() {
    return this;
  }
}
